dart/lib/src/protocol/sentry_stack_trace.dart

```diff
@@ -0,0 +1,33 @@
+import 'sentry_stack_frame.dart';
+
+class SentryStackTrace {
```

Comment:
same comments from SentryStackFrame

Reply:
do we need setters here or should we add a `copyWith` method ?

Reply:
we need a way to mutate the fields, afaik `copyWith` is the most used way on Dart and this is already being used on SentryEvent and other classes, I guess that would be the way to go.

Reply:
If we want a way to mutate the fields, why do we need to mark them as `final`?

Reply:
either we make them public and mutable (losing encapsulation) or we make it immutable and a `copyWith` extension, in Dart, the latter sounds the more correct, do we have another option?

Reply:
✅